nixos/modules/services/desktops/pipewire.nix

```diff
@@ -49,8 +45,13 @@ in {
         enable = mkEnableOption "JACK audio emulation";
       };
 
-      pulse = {
-        enable = mkEnableOption "PulseAudio emulation";
+      pulse-server = {
```

Comment:
I might be missing something but I don't think we need to / should rename this option.

Reply:
Fair enough.